tox.ini

```diff
@@ -1,5 +1,7 @@
 [tox]
-envlist = py35,py36,py37,py38,lint,check-formatting,manifest
+minversion = 3.4.0
```

Comment:
Why this particular tox version?

Reply:
tox isn't meant to be installed with `pip install -e ".[dev]"` - it's meant to be a global/user-local utility and not installed _inside_ a virtualenv of a project.

Reply:
I would argue that `pip install -e ".[dev]"` should install all tools that a contributor needs to follow the contributing guide.

Anyway, as long as tox isn't called recursively, resulting in an infinite nested env creation loop, I don't see a problem with installing it in a dev env (I do that for like 5y with my projects and never had issues).
Maybe I'm missing some critical implication?
As far as I know, all venv solutions manipulate PATH and the first matching command/package found is used.

Reply:
Unlike `pip install -e ".[dev]"`, `tox` should not be installed in a virtualenv. Tox is meant to manage virtualenvs. If you haven't had a problem before, that's lucky :) All venv solutions manipulate the path, but you're running tox in an already-manipulated path. If a virtualenv for say, py 3.7 was created & active, and `tox` is installed & running inside the activated virtualenv, you (as a maintainer of a project) can't guarantee that a user's/contributor's `tox` will be able to pick up the correct python version for anything other than 3.7. 

```sh
# new virtualenv created
(tmp-tox) ~/D/p/interrogate (master) python -V
Python 3.6.11

(tmp-tox) ~/D/p/interrogate (master) pip install -U pip setuptools
# nothing installed
(tmp-tox) ~/D/p/interrogate (master) pip freeze  
# install tox
(tmp-tox) ~/D/p/interrogate (master) pip install tox

# I have tox globally ...
(tmp-tox) ~/D/p/interrogate (master) which tox
/Users/lynn/.local/bin/tox
# ... so use `pyenv which` to get the one installed
(tmp-tox) ~/D/p/interrogate (master) pyenv  which tox
/Users/lynn/.pyenv/versions/tmp-tox/bin/tox

# list which envs are defined
(tmp-tox) ~/D/p/interrogate (master) $(pyenv which tox) -l
py35
py36
py37
py38
lint
check-formatting
manifest

# run the py38 env
(tmp-tox) ~/D/p/interrogate (master) $(pyenv which tox) -e py38
.package recreate: /Users/lynn/Dev/personal/interrogate/.tox/.package
.package installdeps: setuptools>=40.6.0, wheel
py38 create: /Users/lynn/Dev/personal/interrogate/.tox/py38
ERROR: InterpreterNotFound: python3.8
________________________________________________________________________________ summary ________________________________________________________________________________
ERROR:  py38: InterpreterNotFound: python3.8
```

Reply:
Ahhh, thanks for enlightening me 😄 
Guess I missed this since I mostly work on windows, there tox searches at the abs paths of the default installation location.
Where as your example shows, that on linux it looks relative to the current python location.
Guess I got some `reqirements_dev.txt`  files to change in my repos 😅 
And I dropped the commit ofc

Reply:
🙏 thank you :)

Reply:
I thank YOU for teaching me something new 😸